src/tests/testRock.cc

```diff
@@ -97,14 +103,24 @@ TestRock::setUp()
     if (Ipc::Mem::Segment::BasePath == nullptr)
         Ipc::Mem::Segment::BasePath = ".";
 
+    // diff reduction; TODO: inline
+    finishStartup();
```

Comment:
I plan to inline this call (before merging this PR). Inlining now will make it more difficult to see what the new startup code has changed and what it has preserved. Fortunately, this was the only set of tests with complicated setup changes.

Reply:
Done in commit 6810ed1.